src/applications/gi/containers/Modals.jsx

```diff
@@ -926,24 +926,51 @@ export class Modals extends React.Component {
           visible={this.shouldDisplayModal('calcEnrolled')}
         >
           <h3>Enrollment status</h3>
-          <p>
-            Are you considered a full-time or part-time student by your school?
-            Students attending school less than full-time will get a pro-rated
-            monthly housing allowance. Students attending school exactly ½ time
-            or less won’t get a monthly housing allowance.
-          </p>
-          <p>
-            For more information about MHA increases or decreases, visit{' '}
-            <a
-              title="For more information about MHA increases or decreases click here"
-              href="https://gibill.custhelp.com/app/answers/detail/a_id/1412"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              this page
-            </a>
-            .
-          </p>
+          {environment.isProduction() ? (
+            <div>
+              {' '}
+              <p>
+                Are you considered a full-time or part-time student by your
+                school? Students attending school less than full-time will get a
+                pro-rated monthly housing allowance. Students attending school
+                exactly ½ time or less won’t get a monthly housing allowance.
+              </p>
```

Comment:
Is this paragraph and the enclosing div needed for the PR, since only the next paragraph changes?